text/0000-informational-commands.md

```diff
@@ -0,0 +1,766 @@
+
+Additionally, we might consider here (and people have suggested in discussions in Discord) that we simply make the bare `volta` output the equivalent of `volta list`, with `volta --help` or `volta help` being required for the full help details. In this scenario, we would presumably also include a message to that effect for non-TTY, human-friendly contexts, so:
+
+```sh
+$ volta
+⚡️ Volta 0.5.2
+Currently active tools:
+
+    Node: v8.16.0 (default)
+    Yarn: v1.12.3 (from ~/node-and-yarn/package.json)
+    Tool binaries available:
+        create-react-app, ember, tsc, tsserver
+
+To install a tool in your toolchain, use `volta install`.
+To pin your project's runtime or package manager, use `volta pin`.
+
+See `volta help` for more options!
+```
+
+This would be a substantial change, and require considerably more work, but it might be a nice experience!
+
+## Keep `volta current` as an alias for `volta list`
+
+Rather than deprecating `volta current`, we could make it an alias for `volta list`. The primary downside to doing this is that we would want it to work for *only* the equivalent of *bare* `volta list`, and not support any subcommands. This might be mildly confusing to users, but that possibility of confusion should be balanced against the utility of meeting people's existing expectations.
+
+## Do not add `list`
+
+We could leave the current state of affairs as it is. However, users then have no insight into their toolchain: they must inspect the internals of `~/.volta`  to see what they actually have on their system, and in order to see what version of a tool is going to be used, they have must actually invoke it or run `volta which` and interpret the path.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+- Should the bare command include:
+
+    - all available package binaries?
+    - a subset of package binaries once the number crosses a threshold, with instructions about how to see all?
+    - no package binaries?
+
+    And how should that be presented in `plain` vs. `human` mode? Should there be any differences between `plain` and `human` for the bare command (as in the current design)?
+
+- How should the current version be identified? It is currently marked with `(current @ <path>)`. Should this be `(from <path>)` or some other design?
+
+- Should the command accept flags to narrow the search instead of a positional argument, e.g. `volta list --node`, `volta list --yarn`, `volta list --package=typescript`, etc.?
+
+- How should the built-in version of `npm` be displayed to the user? Options include:
+    - associated with the runtime it ships with, e.g. `v12.2.0 (with npm v6.4.1)`
```

Comment:
This is how we currently show it when you run e.g. `volta install node`, so if possible it would be good to align with that. Also, until we implement the `volta install npm` logic, listing `npm` as a separate packager seems like it has the potential for confusion, implying that the user can manage `npm` themselves.